server/testing/src/main/java/org/apache/james/utils/FakeSmtp.java

```diff
@@ -105,4 +106,14 @@ private RequestSpecification requestSpecification() {
     public SwarmGenericContainer getContainer() {
         return container;
     }
+
+    public void clean() {
+        List<String> ids = given(requestSpecification(), RESPONSE_SPECIFICATION)
```

Comment:
👍
I have a look at fakesmtp repository, i think there is another way to truncate all emails which uses:
- `/api/email/latest` to retrieve the latest email id 
- `/api/email/purge/:id` to delete all emails older than an input id

Could you consider?

Reply:
> `/api/email/purge/:id` to delete all emails older than an input id


Isn't it earlier we would be interested in?

Anyway I'll have a look.